src/matchers/toBeArrayOfSize/index.js

```diff
@@ -0,0 +1,34 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = (actual, expected) => () => `${matcherHint('.not.toBeArrayOfSize', 'received', '')}
+
+Expected value to not be an array of size:
```

Comment:
I think we should be consistent with newlines and follow the pattern used in other matchers: 

```
Expected value to not be an array of size:
  ${printExpected(expected)}
Received:
  ${printReceived(actual.length)}
```

Reply:
Can you please confirm if this is the correct exact string literal for this example so that I can amend the others?

```js
const passMessage = (actual, expected) => () => `Expected value to not be an array of size:
  ${printExpected(expected)}
Received:
  ${printReceived(actual.length)}`;
```

Reply:
Looks good to me 👍